### DISCUSSION TITLE
app.config.ts & layers - is it possible to override value from layer in the project, that is extending?

### DISCUSSION THREAD

**Discussion by AloisSeckar** [nuxt/nuxt#25899] [Questions] [answered]
I have simple `app.config.ts` defined in my layer:
```
export default defineAppConfig({
  textTitle: 'eAplikace - Nuxt [DEMO]'
})
```

I intent to use the value for html title in `useHead`. In the project, which I am extending with my layer, I want to be able to override the default falue with project-specific  `app.config.ts`, for example:
```
export default defineAppConfig({
  textTitle: 'eAplikace - EKR'
})
```

But the value from project is ignored and `eAplikace - Nuxt [DEMO]` is shown in the browser tab title. Is there a way to give a higher precedence to project? I guess I can omit setting it in the layer, but I would like to keep the layer value as default fallback if nothing is set in extending project...

I checked [Merging strategy in docs](https://nuxt.com/docs/guide/directory-structure/app-config#merging-strategy), but it didnt seem to work for my plain string. Or should I turn my config value into an array to mimic the desired behavior? Doesnt sound right.

**Comment by TheAlexLichter** [comment, marked as answer]
No, this should be correct.

See [this StackBlitz](https://stackblitz.com/edit/nuxt-discussions-25899?file=.playground%2Fapp.config.ts,.playground%2Fnuxt.config.ts,app.config.ts,app.vue,nuxt.config.ts)

**Reply by AloisSeckar** [reply]
Hm, weird. But thanks for simple example that it SHOULD work. I already closed my work laptop for today, so I will have to check tomorrow, where is the problem.

Maybe one thing is different in my setup though - I am extending from an npm package. Might this affect the behaviour in any way?

**Reply by AloisSeckar** [reply]
Silly mistake 🤦  I didn't noticed in IDE that my `app.config.ts` is nested one level instead being put in root directory of the extending project. After re-locating, it works as you pointed out for sure.
